pkg/services/hcloud/server/server.go

```diff
@@ -627,7 +1134,7 @@ func (s *Service) getSSHKeys(ctx context.Context) (
 	return caphSSHKeys, hcloudSSHKeys, nil
 }
 
-func (s *Service) getServerImage(ctx context.Context) (*hcloud.Image, error) {
+func (s *Service) getServerImage(ctx context.Context, imageName string) (*hcloud.Image, error) {
```

Comment:
why do you change this function? I don't see the point

Reply:
This change is needed:

```go
	image, err := s.getServerImage(ctx, preRescueOSImage)
```